### ISSUE TITLE
Same input format for roc_auc_score and prc_auc_score

### ISSUE THREAD

**Issue by hhaootian** [deepchem/deepchem#3071]
## 🚀 Feature request

roc_auc_score requires labels (1d array) but [prc_auc_score](https://github.com/deepchem/deepchem/blob/master/deepchem/metrics/score_function.py#L103) requires probabilities (2d array). 
How about converting prc_auc_score to keep the same style?

**Comment by rbharath** [comment]
We follow the format followed by sklearn.metrics here. It probably makes sense to keep the current format since that matches broader community usage
